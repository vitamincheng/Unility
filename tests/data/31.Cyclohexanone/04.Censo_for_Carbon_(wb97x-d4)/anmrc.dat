   23.402000  0.000000E+00
   24.715270  0.531955E+03
   26.673771  0.106391E+04
   40.227248  0.106391E+04
  210.701287  0.531955E+03
  211.387937  0.000000E+00
