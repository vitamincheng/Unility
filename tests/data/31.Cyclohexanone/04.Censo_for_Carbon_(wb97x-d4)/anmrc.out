vitamin-desktop
       +--------------------------------------+
       |              A N M R                 |
       |             S. Grimme                |
       |      Universitaet Bonn, MCTC         |
       |             1989-2019                |
       |            version 3.5.1             |
       |     Sat Feb  9 06:41:57 CET 2019     |
       +--------------------------------------+
       Based on a TurboPascal program written  
       in 1989 which was translated to F90 in  
       2005 and re-activated in 2017.          
       Please cite work employing this code as:
       ANMR Ver. 3.5: An automatic, QC based
       coupled NMR spectra simulation program.
       S. Grimme, Universitaet Bonn, 2019
       S. Grimme, C. Bannwarth, S. Dohm, A. Hansen
       J. Pisarek, P. Pracht, J. Seibert, F. Neese
       Angew. Chem. Int. Ed. 2017, 56, 14763-14769.
       DOI:10.1002/anie.201708266               
 
 
 =============================
    # OMP threads =           8
 =============================
 reading <.anmrrc> for standard data
 Read ORCA from anmrrc
           6   197.950000000000       0.000000000000000E+000           1
1H resonance frequency (-mf <real>)    :  500.00
line width             (-lw <real>)    :    1.00
number of all conformers               : 1
temperature in K                       :  298.15
remove J couplings to OH groups        : T
maximum spin system size in a fragment :10
fragmentation type (0=none,1=at,2=mol) : 2
chemical shift scalings a,b            :    1.00    0.00
spin-spin coupling scal factor         :    1.07
plot offset                            :    0.00
Active nuclei                          :C 
 
 reading from anmr_enso
 conformational energies:           1
   0.000
 conformational RRHO energies:
   0.000
 conformational Gsolv free energies:
   0.000
 conformational free energies:
   0.000
 gi per conformer:
    1.00
 conformational populations calculated by anmr:
   1.000
 conformational populations calculated by enso:
 These are used:
   1.000
   1(   1)   100.0
ensemble average free energy (Eh)     : -309.707328
ensemble entropy (cal/mol K)          :   -0.000
ensemble total free energy (kcal/mol) : -309.707328
 reading rotamer data from  <anmr_rotamer>
 number of unique conformers from CREST   3
 number of unique conformers from enso              1
 conformer      1 averaging over    1 rotamers
 conformer      2 averaging over    2 rotamers
 conformer      2 not in anmr_enso, skipping 
 conformer      3 averaging over    4 rotamers
 conformer      3 not in anmr_enso, skipping 
 average over    1 in CRE 
 ====================================================
         reading J/sigma data for conformer            1
 ====================================================
 reading CONF1/NMR/orcaS.out
 reading CONF1/NMR/orcaJ.out
   #  in coord file  # nucs   delta(ppm)
   1         1         1        24.402
   2         2         1        26.360
   3         3         1        39.914
   4         6         1       210.388
   5         9         1        39.914
   6        10         1        26.360

 MATRIX PRINTED:  rotamer average J (Hz) matrix

           1         2         3         4         5         6

    1   0.00000
    2   0.00000   0.00000
    3   0.00000   0.00000   0.00000
    4   0.00000   0.00000   0.00000   0.00000
    5   0.00000   0.00000   0.00000   0.00000   0.00000
    6   0.00000   0.00000   0.00000   0.00000   0.00000   0.00000
 writing spin system data for this confomer to tmpanmr.1
 =====================================================
             average over conformers
 =====================================================
conformer   1 (normalized) population :   1.000

   #  in coord file  # nucs   delta(ppm)
   1         1         1        24.402  +/-      0.00
   2         2         1        26.360  +/-      0.00
   3         3         1        39.914  +/-      0.00
   4         6         1       210.388  +/-      0.00
   5         9         1        39.914  +/-      0.00
   6        10         1        26.360  +/-      0.00

 MATRIX PRINTED:  conformer average J (Hz) matrix

           1         2         3         4         5         6

    1   0.00000
    2   0.00000   0.00000
    3   0.00000   0.00000   0.00000
    4   0.00000   0.00000   0.00000   0.00000
    5   0.00000   0.00000   0.00000   0.00000   0.00000
    6   0.00000   0.00000   0.00000   0.00000   0.00000   0.00000
 ====================================================
               building sub-systems
 ====================================================
 1:10   max/av. J neglected :   0.000   0.000
 2: 9   max/av. J neglected :   0.000   0.000
 3: 8   max/av. J neglected :   0.000   0.000
 4: 7   max/av. J neglected :   0.000   0.000
 5: 6   max/av. J neglected :   0.000   0.000
 selected scheme            5
 system  largest neglected JAB   nuclei (dummy atoms are negative)
    1         0.000        2  -1
    2         0.000        3  -1
    3         0.000        4  -1
    4         0.000        5  -1
    5         0.000        6  -1
    6         0.000        1  -2
            === FRAGMENTED SYSTEM ===
 ====================================================
      solving (J/sigma) averaged spin Hamiltonian
 ====================================================
          spinsystem            1  with            2  spins
             4 product functions     3 Mt blocks, largest is      2
  2(   2)  3(   1)  1(   1) 
first maxtrix multiply,  sparsity in %    87.500 ...
second maxtrix multiply, sparsity in %    75.000 ...
          spinsystem            2  with            2  spins
             4 product functions     3 Mt blocks, largest is      2
  1(   1)  2(   2)  3(   1) 
first maxtrix multiply,  sparsity in %    87.500 ...
second maxtrix multiply, sparsity in %    75.000 ...
          spinsystem            3  with            2  spins
             4 product functions     3 Mt blocks, largest is      2
  2(   2)  1(   1)  3(   1) 
first maxtrix multiply,  sparsity in %    87.500 ...
second maxtrix multiply, sparsity in %    75.000 ...
          spinsystem            4  with            2  spins
             4 product functions     3 Mt blocks, largest is      2
  2(   2)  1(   1)  3(   1) 
first maxtrix multiply,  sparsity in %    87.500 ...
second maxtrix multiply, sparsity in %    75.000 ...
          spinsystem            5  with            2  spins
             4 product functions     3 Mt blocks, largest is      2
  2(   2)  3(   1)  1(   1) 
first maxtrix multiply,  sparsity in %    87.500 ...
second maxtrix multiply, sparsity in %    75.000 ...
          spinsystem            6  with            2  spins
             4 product functions     3 Mt blocks, largest is      2
  3(   1)  2(   2)  1(   1) 
first maxtrix multiply,  sparsity in %    87.500 ...
second maxtrix multiply, sparsity in %    75.000 ...
 done.
          12  non-zero transitions.
 spectrum on <anmr.dat>
 Range (delta in ppm)    23.4020000000000        211.388000000000     
 Range (delta in Hz)     11701.0000000000        105694.000000000     
 Min/max Int.      )    0.500000000000000     
 computing spectrum ...
 done.
 writing output file ...
 done.
 All done.
